core/src/com/mygdx/game/assets/GameScreenAssets.java

```diff
@@ -38,7 +37,7 @@ public boolean update() {
     return assetManager.update();
   }
 
-  public void loadAssets() {
+  public void loadAssetsAsync() {
```

Comment:
Ta nazwa, to do późniejszych prac? Bo narazie chyba nie jest asynchronicznie nie?

Reply:
Ten Async albo Sync odnosi się do tego, że w jednym z xxxxScreenAssets loadAssets ma na końcu finishLoading(), a to jest synchroniczne. loadAssetAsync nie ma wywołania tej metody i kolejne assety ładowane są przez update(). To trochę boli bo to jest taki stary styl asynchroniczności, który nie polega na callbackach tylko na pollowaniu, ale API jest takie jakie jest.